code/game/machinery/doors/blast_door.dm

```diff
@@ -323,5 +328,64 @@ obj/machinery/door/blast/regular/open
 	icon_state = "shutter1"
 	damage = SHUTTER_CRUSH_DAMAGE
 
+// SUBTYPE: Transparent
+// Not technically a blast door but operates like one. Allows air and light.
+obj/machinery/door/blast/gate
+	name = "thick gate"
+	icon_state_open = "tshutter0"
+	icon_state_opening = "tshutterc0"
+	icon_state_closed = "tshutter1"
+	icon_state_closing = "tshutterc1"
+	icon_state = "tshutter1"
+	damage = SHUTTER_CRUSH_DAMAGE
+	maxhealth = 400
+	block_air_zones = 0
+	opacity = 0
+	istransparent = 1
+
+obj/machinery/door/blast/gate/open
+	icon_state = "tshutter0"
+	density = 0
+
+obj/machinery/door/blast/gate/proc/force_close_transparent()
```

Comment:
To my understanding, force_close already checks to see if the source should be transparent, making this proc unneeded.